libraries/helpers.rb

```diff
@@ -629,7 +629,7 @@ def is_slurm_database_updated?
   config["Scheduling"]["SlurmSettings"]["Database"] != previous_config["Scheduling"]["SlurmSettings"]["Database"]
 end
 
-# load cluster configuration file into node object
+# load shared storages data into node object
 def load_shared_storages_mapping
```

Comment:
can this be moved in the fetch_config now?

Reply:
Changed

Reply:
After making this change, I run a test and it didn't work. The node attribute is not loaded later after I move load_shared_storage_data to fetch_config. I am going to keep it in its original place and let's have a further discussion on where is better to keep the function